modules/UI/videolayout/LargeVideoManager.js

```diff
@@ -136,6 +136,13 @@ export default class LargeVideoManager {
                 showAvatar = true;
             }
 
+            // If audio only mode is enabled, always show the avatar except in
+            // instances where an external url is being shown.
+            if (APP.store.getState()['features/base/conference'].audioOnly
+                && !container.url) {
```

Comment:
To check for shared video or etherpad on large you can use this.state 
https://github.com/jitsi/jitsi-meet/blob/22485606996d0e690bce8c0cbca54567b6bc7af1/modules/UI/videolayout/LargeVideoManager.js#L474-L474
So we have VIDEO_CONTAINER_TYPE, SHARED_VIDEO_CONTAINER_TYPE and ETHERPAD_CONTAINER_TYPE